.github/workflows/go-ci.yml

```
@@ -0,0 +1,35 @@
+name: Go CI
+
+on:
+  push:
+    branches: [ main ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)